### DISCUSSION TITLE
Make it easier to use the get multiple routes with slugs

### DISCUSSION THREAD

**Discussion by RandomGgames** [modrinth/.github#47] [Feature requests]
So currently the "get multiple" routes only allows a list of IDs. This is fine, except when you want to automate updating or downloading a list of mods based on just their URL (which contains the mod slug only). This would require a request for each mod to retrieve the ID which is not ideal, especially with the 300 request limit per minute and the potential for a large amount of mods.

I am requesting that either the get multiple routes can accept IDs and Slugs, not just IDs, or a new route is created which will simply return a list of IDs. Something like "https://api.modrinth.com/v2/project-ids" with the parameter slugs=["mod_1", "mod_2"]. This would return a list of ids, such as [mod_1_id, mod_2_id]. This could then be fed into get multiple routes.

This would significantly reduce the amount of requests I have to make.

IF this is already a feature, this should not be a feature request, but rather a bug report, as the get multiple routes on the docs say they require IDs, not IDs or Slugs.

**Comment by triphora** [comment]
The ideal solution here is just to accept both slugs and IDs in the get multiple route. This was suggested for users in modrinth/labrinth#185 but has yet to be added for users or projects.

**Comment by RandomGgames** [comment]
D: I would love to see this added. I feel like it wouldn't be too hard even on the backend.. and that was suggested a while ago!
